deployment/config.go

```diff
@@ -115,6 +115,10 @@ type TerraformDBSettings struct {
 	DBParameters DBParameters
 	// ClusterIdentifier indicates to point to an existing cluster
 	ClusterIdentifier string `default:""`
+	// DBName specifies the name of the database.
+	// If ClusterIdentifier is not empty, DBName should be set to the name of the database in such cluster.
```

Comment:
Should we enforce this rule during validation somehow to prevent more problems?

Reply:
We could validate that it's not empty, but not that it's set to the correct value. Is that what you mean?

Reply:
Yes, if we require it to be set to something we should at least check that or fail early.